### ISSUE TITLE
Android "Could not find any version that matches com.facebook.react:react-native:0.58.+

### ISSUE THREAD

**Issue by sregg** [Instabug/Instabug-React-Native#246]
### Steps to Reproduce the Problem

Build Android app with Instabug React-Native 8.0.25

### Expected Behavior

Build works.

### Actual Behavior

Build fails with this error:

```
Could not find any version that matches com.facebook.react:react-native:0.58.+.
Versions that do not match:
    0.20.1
    0.20.0
    0.19.1
    0.19.0
    0.18.0
    + 13 more
Searched in the following locations:
    https://dl.google.com/dl/android/maven2/com/facebook/react/react-native/maven-metadata.xml
    https://dl.google.com/dl/android/maven2/com/facebook/react/react-native/
    https://jcenter.bintray.com/com/facebook/react/react-native/maven-metadata.xml
Required by:
    project :instabug-reactnative
```

### Instabug integration code
compile project(':instabug-reactnative')

### SDK Version
8.0.25

### React Native, iOS and Android Versions
react-native: 0.57.7
Android: SDK 28, build tools 28.0.3

Seems to be introduced by this commit: https://github.com/Instabug/Instabug-React-Native/commit/81cdb736a72f706c554ab4cd18748be4bce45caf

**Comment by sregg** [comment]
It's working fine when I change in `instabug-reactnative/android/build.gradle`:

`implementation 'com.facebook.react:react-native:0.58.+'`

to

`implementation 'com.facebook.react:react-native:+'`

or even to

`implementation 'com.facebook.react:react-native:0.57.+'`

**Comment by salmamali** [comment]
@sregg This issue is fixed on version 8.0.26. Please update and feel free to reopen the issue of you are still facing this problem.
